dashboard/src/components/settings/signInMethods/ConfirmProvidersUpdatedSettings/ConfirmProvidersUpdatedSettings.tsx

```diff
@@ -0,0 +1,90 @@
+import { useDialog } from '@/components/common/DialogProvider';
+import { useCurrentWorkspaceAndApplication } from '@/hooks/useCurrentWorkspaceAndApplication';
+import { Alert } from '@/ui/Alert';
+import Button from '@/ui/v2/Button';
+import ArrowSquareOutIcon from '@/ui/v2/icons/ArrowSquareOutIcon';
+import Link from '@/ui/v2/Link';
+import Text from '@/ui/v2/Text';
+import { toastStyleProps } from '@/utils/settings/settingsConstants';
+import { useConfirmProvidersUpdatedMutation } from '@/utils/__generated__/graphql';
+import { useState } from 'react';
+import toast from 'react-hot-toast';
+
+export default function ConfirmProvidersUpdatedSettings() {
+  const { currentApplication } = useCurrentWorkspaceAndApplication();
+  const { openAlertDialog } = useDialog();
+  const [visible, setVisible] = useState(true);
```

Comment:
I think instead of managing this state, we could just refetch the query in the parent component when `confirmProvidersUpdated` succeeds. We can let the parent manage the visibility of this alert.

Reply:
Oh, well. Maybe it's not gonna work because we store this information in `currentApplication`. That's managed in kind of a hacky way.